tests/expectations/tests/replace_template_alias.rs

```diff
@@ -7,7 +7,8 @@
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
 pub struct Rooted<T> {
-    pub ptr: MaybeWrapped<T>,
+    pub ptr: ::std::os::raw::c_int,
```

Comment:
This is a regression. We used to correctly have `MaybeWrapped<T>`, but now we have `c_int`. I don't think we should land this patch until this regression is fixed.